sig-security/self-assessment.md

```diff
@@ -0,0 +1,228 @@
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+
+## Metadata
+
+|                   |                                                              |
+| ----------------- | ------------------------------------------------------------ |
+| Software          | https://github.com/kubeedge/kubeedge                         |
+| Website           | https://kubeedge.io                                          |
+| Security Provider | No                                                           |
+| Languages         | Go                                                           |
+| SBOM              | Check [go.mod](https://github.com/kubeedge/kubeedge/blob/master/go.mod) for libraries, packages, versions used by the project |
+
+### Security links
+
+| Doc                          | url                                                          |
+| ---------------------------- | ------------------------------------------------------------ |
+| Security file                | [SECURITY.md](https://github.com/kubeedge/kubeedge/blob/master/SECURITY.md) |
+| Default and optional configs | [cloudcore](https://github.com/kubeedge/kubeedge/blob/master/pkg/apis/componentconfig/cloudcore/v1alpha1/default.go)<br />[edgecore](https://github.com/kubeedge/kubeedge/blob/master/pkg/apis/componentconfig/edgecore/v1alpha1/default.go) |
+
+## Overview
+
+KubeEdge is an open source system for extending native containerized application orchestration capabilities to hosts at the edge. It's built upon Kubernetes and provides fundamental infrastructure support for networking, application deployment and metadata synchronization between cloud and edge.
+
+Since joining CNCF, KubeEdge has attracted more than [1000+ Contributors](https://kubeedge.devstats.cncf.io/d/18/overall-project-statistics-table?orgId=1) from 80+ different Organizations with 3,900+ Commits; got 5,600+ Stars on Github and 1,600+ Forks. It has been adopted by China Mobile, China Telecom, Raisecom, WoCloud, Xinghai IoT, KubeSphere, HUAWEI CLOUD, Harmony Cloud, DaoCloud, SAIC Motor, SF Express, etc.
+
+### Background
+
+As 5G and AI technologies grow, enterprises are increasingly demanding intelligent upgrade, and the application scenarios of edge computing are becoming more and more extensive. It is challenging to realize unified management and control of edge computing resources and cloud-edge synergy, such as how to run intelligent applications and algorithms on edge devices with limited resources(e.g. cameras and drones), how to solve the problems caused by the access of mass heterogeneous edge devices in intelligent transportation and intelligent energy, and how to ensure high reliability of services in off line scenarios.

```

Comment:
i'm wondering whether the intelligent applications are running on edge devices or on the edge nodes with computing resources.

Reply:
I think `edge devices` would be better in this context.